<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Publications</title>

    <!-- Bootstrap Core CSS -->
    <link href="../css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="../css/modern-business.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="../font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">

    <!-- Using all of this for a bootstrap datatables -->
    <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet">   
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <link rel="stylesheet" 
    href="http://cdn.datatables.net/1.10.2/css/jquery.dataTables.min.css"></style>
    <script type="text/javascript" 
    src="http://cdn.datatables.net/1.10.2/js/jquery.dataTables.min.js"></script>
    <script type="text/javascript" 
    src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>

    </head>

    <body>
    <body>
        <!-- Page Content -->
        <div class="container">

            <!-- Page Heading/Breadcrumbs -->
            <div class="row">
                <div class="col-lg-12">
                    <h1 class="page-header">Publications
                        <small></small>
                    </h1>
                </div>
            </div>
            <!-- /.row -->
            <div class="row">
            <div class="col-lg-12">
        <!-- Navigation bar is from a script I made that pastes text -->
        <script type="text/javascript" src="/js/header.js"></script>
        <table id="example" class="display" cellspacing="0" width="100%">
        <thead>
            <tr>
            <td>Authors</td>
            <td>Title</td>
            <td>Year</td>
            <td>Source title</td>
            <td>DOI</td>
            <td>Document Type</td>
            </tr>
        </thead>
        <tbody>
            <tr>
            <td>Li L., Briskine R., Schaefer R., Schnable P.S., Myers C.L., Flagel L.E., Springer N.M., Muehlbauer G.J.</td>
            <td>Co-expression network analysis of duplicate genes in maize (Zea mays L.) reveals no subgenome bias</td>
            <td>2016</td>
            <td>BMC Genomics</td>
            <td>10.1186/s12864-016-3194-0</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Styles E.B., Founk K.J., Zamparo L.A., Sing T.L., Altintas D., Ribeyre C., Ribaud V., Rougemont J., Mayhew D., Costanzo M., Usaj M., Verster A.J., Koch E.N., Novarina D., Graf M., Luke B., Muzi-Falconi M., Myers C.L., Mitra R.D., Shore D., Brown G.W., Zhang Z., Boone C., Andrews B.J.</td>
            <td>Exploring Quantitative Yeast Phenomics with Single-Cell Analysis of DNA Damage Foci</td>
            <td>2016</td>
            <td>Cell Systems</td>
            <td>10.1016/j.cels.2016.08.008</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Costanzo M., VanderSluis B., Koch E.N., Baryshnikova A., Pons C., Tan G., Wang W., Usaj M., Hanchard J., Lee S.D., Pelechano V., Styles E.B., Billmann M., Van Leeuwen J., Van Dyk N., Lin Z.-Y., Kuzmin E., Nelson J., Piotrowski J.S., Srikumar T., Bahr S., Chen Y., Deshpande R., Kurat C.F., Li S.C., Li Z., Usaj M.M., Okada H., Pascoe N., Luis B.-J.S., Sharifpoor S., Shuteriqi E., Simpkins S.W., Snider J., Suresh H.G., Tan Y., Zhu H., Malod-Dognin N., Janjic V., Przulj N., Troyanskaya O.G., Stagljar I., Xia T., Ohya Y., Gingras A.-C., Raught B., Boutros M., Steinmetz L.M., Moore C.L., Rosebrock A.P., Caudy A.A., Myers C.L., Andrews B., Boone C.</td>
            <td>A global genetic interaction network maps a wiring diagram of cellular function</td>
            <td>2016</td>
            <td>Science</td>
            <td>10.1126/science.aaf1420</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Andrews B.J., Marian Walhout A.J., Iyengar R., Apweiler R., Ardlie K., Azeloglu E.U., Birtwistle M.R., Coon J.J., Dolinski K., Fan T., FitzGerald G.A., Gavin A.-C., Gingras A.-C., Gough N.R., Hoffmann A., Lee M.J., Loew L.M., CraigMak H., Murphy R.C., Myers C., Snyder M.P., Sorger P.K., Stolovitzky G., Subramaniam S., Taipale M., Travé G., Troyanskaya O.G., Uhlen M., Vidal M.</td>
            <td>Quantitative human cell encyclopedia</td>
            <td>2016</td>
            <td>Science Signaling</td>
            <td>10.1126/scisignal.aah4406</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Mitra A.K., Stessman H.A.F., Schaefer R.J., Wang W., Myers C.L., Van Ness B.G., Beiraghi S.</td>
            <td>Fine-mapping of 18q21.1 locus identifies single nucleotide polymorphisms associated with nonsyndromic cleft lip with or without cleft palate</td>
            <td>2016</td>
            <td>Frontiers in Genetics</td>
            <td>10.3389/fgene.2016.00088</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Dickinson Q., Bottoms S., Hinchman L., McIlwain S., Li S., Myers C.L., Boone C., Coon J.J., Hebert A., Sato T.K., Landick R., Piotrowski J.S.</td>
            <td>Mechanism of imidazolium ionic liquids toxicity in Saccharomyces cerevisiae and rational engineering of a tolerant, xylose-fermenting strain</td>
            <td>2016</td>
            <td>Microbial Cell Factories</td>
            <td>10.1186/s12934-016-0417-7</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Serate J., Xie D., Pohlmann E., Donald C., Jr., Shabani M., Hinchman L., Higbee A., McGee M., La Reau A., Klinger G.E., Li S., Myers C.L., Boone C., Bates D.M., Cavalier D., Eilert D., Oates L.G., Sanford G., Sato T.K., Dale B., Landick R., Piotrowski J., Ong R.G., Zhang Y.</td>
            <td>Controlling microbial contamination during hydrolysis of AFEX-pretreated corn stover and switchgrass: Effects on hydrolysate composition, microbial response and fermentation</td>
            <td>2015</td>
            <td>Biotechnology for Biofuels</td>
            <td>10.1186/s13068-015-0356-2</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Becker J.R., Pons C., Nguyen H.D., Costanzo M., Boone C., Myers C.L., Bielinsky A.-K.</td>
            <td>Genetic Interactions Implicating Postreplicative Repair in Okazaki Fragment Processing</td>
            <td>2015</td>
            <td>PLoS Genetics</td>
            <td>10.1371/journal.pgen.1005659</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Piotrowski J.S., Okada H., Lu F., Li S.C., Hinchman L., Ranjan A., Smith D.L., Higbee A.J., Ulbrich A., Coon J.J., Deshpande R., Bukhman Y.V., McIlwain S., Ong I.M., Myers C.L., Boonec C., Landicka R., Ralpha J., Kabbaged M., Ohyab Y.</td>
            <td>Plant-derived antifungal agent poacic acid targets β-1,3-glucan</td>
            <td>2015</td>
            <td>Proceedings of the National Academy of Sciences of the United States of America</td>
            <td>10.1073/pnas.1410400112</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Schaefer R.J., Briskine R., Springer N.M., Myers C.L.</td>
            <td>Correction: Discovering functional modules across diverse maize transcriptomes using COB, the Co-expression Browser</td>
            <td>2015</td>
            <td>PLoS ONE</td>
            <td>10.1371/journal.pone.0120222</td>
            <td>Erratum</td>
            </tr>
            <tr>
            <td>Pons C., Costanzo M., Boone C., Myers C.L.</td>
            <td>Computational paradigms for analyzing genetic interaction networks</td>
            <td>2015</td>
            <td>Systems Genetics: Linking Genotypes and Phenotypes</td>
            <td>10.1017/CBO9781139012751.002</td>
            <td>Book Chapter</td>
            </tr>
            <tr>
            <td>MacNeil L.T., Pons C., Arda H.E., Giese G.E., Myers C.L., Walhout A.J.M.</td>
            <td>Transcription Factor Activity Mapping of a Tissue-Specific in Vivo Gene Regulatory Network</td>
            <td>2015</td>
            <td>Cell Systems</td>
            <td>10.1016/j.cels.2015.08.003</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Piotrowski J.S., Simpkins S.W., Li S.C., Deshpande R., McIlwain S.J., Ong I.M., Myers C.L., Boone C., Andersen R.J.</td>
            <td>Chemical genomic profiling via barcode sequencing to predict compound mode of action</td>
            <td>2015</td>
            <td>Methods in Molecular Biology</td>
            <td>10.1007/978-1-4939-2269-7_23</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Bansal M., Yang J., Karan C., Menden M.P., Costello J.C., Tang H., Xiao G., Li Y., Allen J., Zhong R., Chen B., Kim M., Wang T., Heiser L.M., Realubit R., Mattioli M., Alvarez M.J., Shen Y., Gallahan D., Singer D., Saez-Rodriguez J., Xie Y., Stolovitzky G., Califano A., Abbuehl J.-P., Altman R.B., Balcome S., Bell A., Bender A., Berger B., Bernard J., Bieberich A.A., Borboudakis G., Chan C., Chen T.-H., Choi J., Coelho L.P., Creighton C.J., Dampier W., Davisson V.J., Deshpande R., Diao L., Di Camillo B., Dundar M., Ertel A., Goswami C.P., Gottlieb A., Gould M.N., Goya J., Grau M., Gray J.W., Hejase H.A., Hoffmann M.F., Homicsko K., Homilius M., Hwang W., Ijzerman A.P., Kallioniemi O., Karacali B., Kaski S., Kim J., Krishnan A., Lee J., Lee Y.-S., Lenselink E.B., Lenz P., Li L., Li J., Liang H., Mpindi J.-P., Myers C.L., Newton M.A., Overington J.P., Parkkinen J., Prill R.J., Peng J., Pestell R., Qiu P., Rajwa B., Sadanandam A., Sambo F., Sridhar A., Sun W., Toffolo G.M., Tozeren A., Troyanskaya O.G., Tsamardinos I., Van Vlijmen H.W.T., Wang W., Wegner J.K., Wennerberg K., Van Westen G.J.P., Xia T., Yang Y., Yao V., Yuan Y., Zeng H., Zhang S., Zhao J., Zhou J.</td>
            <td>A community computational challenge to predict the activity of pairs of compounds</td>
            <td>2014</td>
            <td>Nature Biotechnology</td>
            <td>10.1038/nbt.3052</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Costello J.C., Heiser L.M., Georgii E., Gönen M., Menden M.P., Wang N.J., Bansal M., Ammad-Ud-Din M., Hintsanen P., Khan S.A., Mpindi J.-P., Kallioniemi O., Honkela A., Aittokallio T., Wennerberg K., Collins J.J., Gallahan D., Singer D., Saez-Rodriguez J., Kaski S., Gray J.W., Stolovitzky G., Abbuehl J.-P., Allen J., Altman R.B., Balcome S., Battle A., Bender A., Berger B., Bernard J., Bhattacharjee M., Bhuvaneshwar K., Bieberich A.A., Boehm F., Califano A., Chan C., Chen B., Chen T.-H., Choi J., Coelho L.P., Cokelaer T., Collins J.C., Creighton C.J., Cui J., Dampier W., Davisson V.J., De Baets B., Deshpande R., DiCamillo B., Dundar M., Duren Z., Ertel A., Fan H., Fang H., Gauba R., Gottlieb A., Grau M., Gusev Y., Ha M.J., Han L., Harris M., Henderson N., Hejase H.A., Homicsko K., Hou J.P., Hwang W., IJzerman A.P., Karacali B., Keles S., Kendziorski C., Kim J., Kim M., Kim Y., Knowles D.A., Koller D., Lee J., Lee J.K., Lenselink E.B., Li B., Li B., Li J., Liang H., Ma J., Madhavan S., Mooney S., Myers C.L., Newton M.A., Overington J.P., Pal R., Peng J., Pestell R., Prill R.J., Qiu P., Rajwa B., Sadanandam A., Sambo F., Shin H., Song J., Song L., Sridhar A., Stock M., Sun W., Ta T., Tadesse M., Tan M., Tang H., Theodorescu D., Toffolo G.M., Tozeren A., Trepicchio W., Varoquaux N., Vert J.-P., Waegeman W., Walter T., Wan Q., Wang D., Wang W., Wang Y., Wang Z., Wegner J.K., Wu T., Xia T., Xiao G., Xie Y., Xu Y., Yang J., Yuan Y., Zhang S., Zhang X.-S., Zhao J., Zuo C., Van Vlijmen H.W.T., Van Westen G.J.P.</td>
            <td>A community effort to assess and improve drug sensitivity prediction algorithms</td>
            <td>2014</td>
            <td>Nature Biotechnology</td>
            <td>10.1038/nbt.2877</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Abbey D.A., Funt J., Lurie-Weinberger M.N., Thompson D.A., Regev A., Myers C.L., Berman J.</td>
            <td>YMAP: A pipeline for visualization of copy number variation and loss of heterozygosity in eukaryotic pathogens</td>
            <td>2014</td>
            <td>Genome Medicine</td>
            <td>10.1186/s13073-014-0100-8</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Brown J.C.S., Nelson J., Vandersluis B., Deshpande R., Butts A., Kagan S., Polacheck I., Krysan D.J., Myers C.L., Madhani H.D.</td>
            <td>Unraveling the biology of a fungal meningitis pathogen using chemical genetics</td>
            <td>2014</td>
            <td>Cell</td>
            <td>10.1016/j.cell.2014.10.044</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Stessman H.A., Lulla A., Xia T., Mitra A., Harding T., Mansoor A., Myers C.L., Van Ness B.G., Dolloff N.G.</td>
            <td>High-throughput drug screening identifies compounds and molecular strategies for targeting proteasome inhibitor-resistant multiple myeloma</td>
            <td>2014</td>
            <td>Leukemia</td>
            <td>10.1038/leu.2014.214</td>
            <td>Letter</td>
            </tr>
            <tr>
            <td>Wyche T.P., Piotrowski J.S., Hou Y., Braun D., Deshpande R., McIlwain S., Ong I.M., Myers C.L., Guzei I.A., Westler W.M., Andes D.R., Bugni T.S.</td>
            <td>Forazoline A: Marine-Derived Polyketide with Antifungal in Vivo Efficacy</td>
            <td>2014</td>
            <td>Angewandte Chemie - International Edition</td>
            <td>10.1002/anie.201405990</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Stessman H.A.F., Lulla A., Xia T., Mitra A., Harding T., Mansoor A., Myers C.L., Van Ness B.G., Dolloff N.G.</td>
            <td>High-throughput drug screening identifies compounds and molecular strategies for targeting proteasome inhibitor-resistant multiple myeloma</td>
            <td>2014</td>
            <td>Leukemia</td>
            <td>10.1038/leu.2014.214</td>
            <td>Article in Press</td>
            </tr>
            <tr>
            <td>Schaefer R.J., Briskine R., Springer N.M., Myers C.L.</td>
            <td>Discovering functional modules across diverse maize transcriptomes using COB, the co-expression browser</td>
            <td>2014</td>
            <td>PLoS ONE</td>
            <td>10.1371/journal.pone.0099193</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Fung S.-Y., Sofiyev V., Schneiderman J., Hirschfeld A.F., Victor R.E., Woods K., Piotrowski J.S., Deshpande R., Li S.C., De Voogd N.J., Myers C.L., Boone C., Andersen R.J., Turvey S.E.</td>
            <td>Unbiased screening of marine sponge extracts for antiinflammatory agents combined with chemical genomics identifies girolline as an inhibitor of protein synthesis</td>
            <td>2014</td>
            <td>ACS Chemical Biology</td>
            <td>10.1021/cb400740</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Fung S.-Y., Sofiyev V., Schneiderman J., Hischfeld A.F., Victor R.E., Woods K., Piotrowski J.S., Deshpande R., Li S.C., De Voogd N.J., Myers C.L., Boone C., Andersen R.J., Turvey S.E.</td>
            <td>Unbiased screening of marine sponge extracts for antiinflammatory agents combined with chemical genomics identifies girolline as an inhibitor of protein synthesis</td>
            <td>2014</td>
            <td>ACS Chemical Biology</td>
            <td>10.1021/cb400740c</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Kim Y., Tsuda K., Igarashi D., Hillmer R.A., Sakakibara H., Myers C.L., Katagiri F.</td>
            <td>Mechanisms underlying robustness and tunability in a plant immune signaling network</td>
            <td>2014</td>
            <td>Cell Host and Microbe</td>
            <td>10.1016/j.chom.2013.12.002</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Sekhon R.S., Briskine R., Hirsch C.N., Myers C.L., Springer N.M., Buell C.R., De Leon N., Kaeppler S.M.</td>
            <td>Correction: Maize gene atlas developed by RNA sequencing and comparative evaluation of transcriptomes based on RNA sequencing and microarrays (PLoS ONE)</td>
            <td>2014</td>
            <td>PLoS ONE</td>
            <td>10.1371/annotation/0444d495-9231-4097-abe0-4750b9045971</td>
            <td>Erratum</td>
            </tr>
            <tr>
            <td>DiPrima S., Haarer B., Viggiano S., Pons C., Myers C.L., Amberg D.C.</td>
            <td>Linking genetics to structural biology: Complex heterozygosity screening with actin alanine scan alleles identifies functionally related surfaces on yeast actin</td>
            <td>2014</td>
            <td>G3: Genes, Genomes, Genetics</td>
            <td>10.1534/g3.114.012054</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Kuzmin E., Sharifpoor S., Baryshnikova A., Costanzo M., Myers C.L., Andrews B.J., Boone C.</td>
            <td>Synthetic genetic array analysis for global mapping of genetic networks in yeast</td>
            <td>2014</td>
            <td>Methods in Molecular Biology</td>
            <td>10.1007/978-1-4939-1363-3_10</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Bass J.I.F., Diallo A., Nelson J., Soto J.M., Myers C.L., Walhout A.J.M.</td>
            <td>Corrigendum: Using networks to measure similarity between genes: Association index selection</td>
            <td>2014</td>
            <td>Nature Methods</td>
            <td>10.1038/nmeth0314-349c</td>
            <td>Erratum</td>
            </tr>
            <tr>
            <td>VanderSluis B., Hess D.C., Pesyna C., Krumholz E.W., Syed T., Szappanos B., Nislow C., Papp B., Troyanskaya O.G., Myers C.L., Caudy A.A.</td>
            <td>Broad metabolic sensitivity profiling of a prototrophic yeast deletion collection</td>
            <td>2014</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2014-15-4-r64</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Bass J.I.F., Diallo A., Nelson J., Soto J.M., Myers C.L., Walhout A.J.M.</td>
            <td>Using networks to measure similarity between genes: Association index selection</td>
            <td>2013</td>
            <td>Nature Methods</td>
            <td>10.1038/nmeth.2728</td>
            <td>Review</td>
            </tr>
            <tr>
            <td>Deshpande R., Asiedu M.K., Klebig M., Sutor S., Kuzmin E., Nelson J., Piotrowski J., Shin S.H., Yoshida M., Costanzo M., Boone C., Wigle D.A., Myers C.L.</td>
            <td>A comparative genomic approach for identifying synthetic lethal interactions in human cancer</td>
            <td>2013</td>
            <td>Cancer Research</td>
            <td>10.1158/0008-5472.CAN-12-3956</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Costanzo M., Baryshnikova A., VanderSluis B., Andrews B., Myers C.L., Boone C.</td>
            <td>Genetic Networks</td>
            <td>2013</td>
            <td>Handbook of Systems Biology</td>
            <td>10.1016/B978-0-12-385944-0.00006-X</td>
            <td>Book Chapter</td>
            </tr>
            <tr>
            <td>Baryshnikova A., Costanzo M., Myers C.L., Andrews B., Boone C.</td>
            <td>Genetic interaction networks: Toward an understanding of heritability</td>
            <td>2013</td>
            <td>Annual Review of Genomics and Human Genetics</td>
            <td>10.1146/annurev-genom-082509-141730</td>
            <td>Review</td>
            </tr>
            <tr>
            <td>Eichten S.R., Briskine R., Song J., Li Q., Swanson-Wagner R., Hermanson P.J., Waters A.J., Starr E., West P.T., Tiffin P., Myers C.L., Vaughn M.W., Springer N.M.</td>
            <td>Epigenetic and Genetic Influences on DNA Methylation Variation in Maize Populations</td>
            <td>2013</td>
            <td>Plant Cell</td>
            <td>10.1105/tpc.113.114793</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Reece-Hoyes J.S., Pons C., Diallo A., Mori A., Shrestha S., Kadreppa S., Nelson J., DiPrima S., Dricot A., Lajoie B.R., Ribeiro P.S.M., Weirauch M.T., Hill D.E., Hughes T.R., Myers C.L., Walhout A.J.M.</td>
            <td>Extensive rewiring and complex evolutionary dynamics in a C.elegans multiparameter transcription factor network</td>
            <td>2013</td>
            <td>Molecular Cell</td>
            <td>10.1016/j.molcel.2013.05.018</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Deshpande R., VanderSluis B., Myers C.L.</td>
            <td>Comparison of Profile Similarity Measures for Genetic Interaction Networks</td>
            <td>2013</td>
            <td>PLoS ONE</td>
            <td>10.1371/journal.pone.0068664</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Wagih O., Usaj M., Baryshnikova A., VanderSluis B., Kuzmin E., Costanzo M., Myers C.L., Andrews B.J., Boone C.M., Parts L.</td>
            <td>SGAtools: One-stop analysis and visualization of array-based genetic interaction screens.</td>
            <td>2013</td>
            <td>Nucleic acids research</td>
            <td>10.1093/nar/gkt400</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Da Nguyen H., Becker J., Thu Y.M., Costanzo M., Koch E.N., Smith S., Myung K., Myers C.L., Boone C., Bielinsky A.-K.</td>
            <td>Unligated Okazaki Fragments Induce PCNA Ubiquitination and a Requirement for Rad59-Dependent Replication Fork Progression</td>
            <td>2013</td>
            <td>PLoS ONE</td>
            <td>10.1371/journal.pone.0066379</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Stessman H.A.F., Baughn L.B., Sarver A., Xia T., Deshpande R., Mansoor A., Walsh S.A., Sunderland J.J., Dolloff N.G., Linden M.A., Zhan F., Janz S., Myers C.L., Van Ness B.G.</td>
            <td>Profiling bortezomib resistance identifies secondary therapies in a mouse myeloma model</td>
            <td>2013</td>
            <td>Molecular Cancer Therapeutics</td>
            <td>10.1158/1535-7163.MCT-12-1151</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Sekhon R.S., Briskine R., Hirsch C.N., Myers C.L., Springer N.M., Buell C.R., de Leon N., Kaeppler S.M.</td>
            <td>Maize Gene Atlas Developed by RNA Sequencing and Comparative Evaluation of Transcriptomes Based on RNA Sequencing and Microarrays</td>
            <td>2013</td>
            <td>PLoS ONE</td>
            <td>10.1371/journal.pone.0061005</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Colak R., Kim T.H., Michaut M., Sun M., Irimia M., Bellay J., Myers C.L., Blencowe B.J., Kim P.M.</td>
            <td>Distinct Types of Disorder in the Human Proteome: Functional Implications for Alternative Splicing</td>
            <td>2013</td>
            <td>PLoS Computational Biology</td>
            <td>10.1371/journal.pcbi.1003030</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Makarevitch I., Eichten S.R., Briskine R., Waters A.J., Danilevskaya O.N., Meeley R.B., Myers C.L., Vaughn M.W., Springer N.M.</td>
            <td>Genomic distribution of maize facultative heterochromatin marked by trimethylation of H3K27</td>
            <td>2013</td>
            <td>Plant Cell</td>
            <td>10.1105/tpc.112.106427</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Alver B., Jauert P.A., Brosnan L., O&#39;Hehir M., VanderSluis B., Myers C.L., Kirkpatrick D.T.</td>
            <td>A whole genome screen for minisatellite stability genes in stationary-phase yeast cells</td>
            <td>2013</td>
            <td>G3: Genes, Genomes, Genetics</td>
            <td>10.1534/g3.112.005397</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Baryshnikova A., VanderSluis B., Costanzo M., Myers C.L., Cha R.S., Andrews B., Boone C.</td>
            <td>Global linkage map connects meiotic centromere function to chromosome size in budding yeast</td>
            <td>2013</td>
            <td>G3: Genes, Genomes, Genetics</td>
            <td>10.1534/g3.113.007377</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Swanson-Wagner R., Briskine R., Schaefer R., Hufford M.B., Ross-Ibarra J., Myers C.L., Tiffin P., Springer N.M.</td>
            <td>Reshaping of the maize transcriptome by domestication</td>
            <td>2012</td>
            <td>Proceedings of the National Academy of Sciences of the United States of America</td>
            <td>10.1073/pnas.1201961109</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Koch E.N., Costanzo M., Bellay J., Deshpande R., Chatfield-Reed K., Chua G., D&#39;Urso G., Andrews B.J., Boone C., Myers C.L.</td>
            <td>Conserved rules govern genetic interaction degree across species</td>
            <td>2012</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2012-13-7-r57</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Sharifpoor S., Van Dyk D., Costanzo M., Baryshnikova A., Friesen H., Douglas A.C., Youn J.-Y., VanderSluis B., Myers C.L., Papp B., Boone C., Andrews B.J.</td>
            <td>Functional wiring of the yeast kinome revealed by global analysis of genetic network motifs</td>
            <td>2012</td>
            <td>Genome Research</td>
            <td>10.1101/gr.129213.111</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Davidson M.B., Katou Y., Keszthelyi A., Sing T.L., Xia T., Ou J., Vaisica J.A., Thevakumaran N., Marjavaara L., Myers C.L., Chabes A., Shirahige K., Brown G.W.</td>
            <td>Endogenous DNA replication stress results in expansion of dNTP pools and a mutator phenotype</td>
            <td>2012</td>
            <td>EMBO Journal</td>
            <td>10.1038/emboj.2011.485</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Bellay J., Michaut M., Kim T., Han S., Colak R., Myers C.L., Kim P.M.</td>
            <td>An omics perspective of protein disorder</td>
            <td>2012</td>
            <td>Molecular BioSystems</td>
            <td>10.1039/c1mb05235g</td>
            <td>Review</td>
            </tr>
            <tr>
            <td>Adams D., Berger B., Harismendy O., Huttenhower C., Liu S.X., Myers C., Oshlack A., Rinn J., Walhout M.</td>
            <td>As we come to the end of 2011, several members of the Genome Biology Editorial Board give their views on the state of play in genomics</td>
            <td>2011</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2011-12-12-137</td>
            <td>Note</td>
            </tr>
            <tr>
            <td>Reece-Hoyes J.S., Diallo A., Lajoie B., Kent A., Shrestha S., Kadreppa S., Pesyna C., Dekker J., Myers C.L., Walhout A.J.M.</td>
            <td>Enhanced yeast one-hybrid assays for high-throughput gene-centered regulatory network mapping</td>
            <td>2011</td>
            <td>Nature Methods</td>
            <td>10.1038/nmeth.1748</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Adams D.J., Berger B., Harismendy O., Huttenhower C., Liu X.S., Myers C.L., Oshlack A., Rinn J.L., Walhout A.J.</td>
            <td>Genomics in 2011: challenges and opportunities.</td>
            <td>2011</td>
            <td>Genome biology</td>
            <td> </td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Cokol M., Chua H.N., Tasan M., Mutlu B., Weinstein Z.B., Suzuki Y., Nergiz M.E., Costanzo M., Baryshnikova A., Giaever G., Nislow C., Myers C.L., Andrews B.J., Boone C., Roth F.P.</td>
            <td>Systematic exploration of synergistic drug pairs</td>
            <td>2011</td>
            <td>Molecular Systems Biology</td>
            <td>10.1038/msb.2011.71</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Williams D.E., Dalisay D.S., Patrick B.O., Matainaho T., Andrusiak K., Deshpande R., Myers C.L., Piotrowski J.S., Boone C., Yoshida M., Andersen R.J.</td>
            <td>Padanamides A and B, highly modified linear tetrapeptides produced in culture by a Streptomyces sp. Isolated from a marine sediment</td>
            <td>2011</td>
            <td>Organic Letters</td>
            <td>10.1021/ol2014494</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Bellay J., Atluri G., Sing T.L., Toufighi K., Costanzo M., Ribeiro P.S.M., Pandey G., Baller J., VanderSluis B., Michaut M., Han S., Kim P., Brown G.W., Andrews B.J., Boone C., Kumar V., Myers C.L.</td>
            <td>Putting genetic interactions in context through a global modular decomposition</td>
            <td>2011</td>
            <td>Genome Research</td>
            <td>10.1101/gr.117176.110</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Szappanos B., Kovács K., Szamecz B., Honti F., Costanzo M., Baryshnikova A., Gelius-Dietrich G., Lercher M.J., Jelasity M., Myers C.L., Andrews B.J., Boone C., Oliver S.G., Pál C., Papp B.</td>
            <td>An integrated approach to characterize genetic interaction networks in yeast metabolism</td>
            <td>2011</td>
            <td>Nature Genetics</td>
            <td>10.1038/ng.846</td>
            <td>Conference Paper</td>
            </tr>
            <tr>
            <td>Li Z., Vizeacoumar F.J., Bahr S., Li J., Warringer J., Vizeacoumar F.S., Min R., Vandersluis B., Bellay J., Devit M., Fleming J.A., Stephens A., Haase J., Lin Z.-Y., Baryshnikova A., Lu H., Yan Z., Jin K., Barker S., Datti A., Giaever G., Nislow C., Bulawa C., Myers C.L., Costanzo M., Gingras A.-C., Zhang Z., Blomberg A., Bloom K., Andrews B., Boone C.</td>
            <td>Systematic exploration of essential yeast gene function with temperature-sensitive mutants</td>
            <td>2011</td>
            <td>Nature Biotechnology</td>
            <td>10.1038/nbt.1832</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Bellay J., Han S., Michaut M., Kim T., Costanzo M., Andrews B.J., Boone C., Bader G.D., Myers C.L., Kim P.M.</td>
            <td>Bringing order to protein disorder through comparative genomics and genetic interactions</td>
            <td>2011</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2011-12-2-r14</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Costanzo M., Baryshnikova A., Myers C.L., Andrews B., Boone C.</td>
            <td>Charting the genetic interaction map of a cell</td>
            <td>2011</td>
            <td>Current Opinion in Biotechnology</td>
            <td>10.1016/j.copbio.2010.11.001</td>
            <td>Review</td>
            </tr>
            <tr>
            <td>Michaut M., Baryshnikova A., Costanzo M., Myers C.L., Andrews B.J., Boone C., Bader G.D.</td>
            <td>Protein complexes are central in the yeast genetic landscape</td>
            <td>2011</td>
            <td>PLoS Computational Biology</td>
            <td>10.1371/journal.pcbi.1001092</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Deshpande R., Sharma S., Verfaillie C.M., Hu W.-S., Myers C.L.</td>
            <td>A scalable approach for discovering conserved active subnetworks across species</td>
            <td>2010</td>
            <td>PLoS Computational Biology</td>
            <td>10.1371/journal.pcbi.1001028</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Fang G., Kuang R., Pandey G., Steinbach M., Myers C.L., Kumar V.</td>
            <td>Subspace differential coexpression analysis: Problem definition and a general approach</td>
            <td>2010</td>
            <td>Pacific Symposium on Biocomputing 2010, PSB 2010</td>
            <td> </td>
            <td>Conference Paper</td>
            </tr>
            <tr>
            <td>Baryshnikova A., Costanzo M., Kim Y., Ding H., Koh J., Toufighi K., Youn J.-Y., Ou J., San Luis B.-J., Bandyopadhyay S., Hibbs M., Hess D., Gingras A.-C., Bader G.D., Troyanskaya O.G., Brown G.W., Andrews B., Boone C., Myers C.L.</td>
            <td>Quantitative analysis of fitness and genetic interactions in yeast on a genome scale</td>
            <td>2010</td>
            <td>Nature Methods</td>
            <td>10.1038/nmeth.1534</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Vandersluis B., Bellay J., Musso G., Costanzo M., Papp B., Vizeacoumar F.J., Baryshnikova A., Andrews B., Boone C., Myers C.L.</td>
            <td>Genetic interactions reveal the evolutionary trajectories of duplicate genes</td>
            <td>2010</td>
            <td>Molecular Systems Biology</td>
            <td>10.1038/msb.2010.82</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Baryshnikova A., Costanzo M., Dixon S., Vizeacoumar F.J., Myers C.L., Andrews B., Boone C.</td>
            <td>Synthetic genetic array (SGA) analysis in Saccharomyces cerevisiae and schizosaccharomyces pombe</td>
            <td>2010</td>
            <td>Methods in Enzymology</td>
            <td>10.1016/S0076-6879(10)70007-0</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Pandey G., Zhang B., Chang A.N., Myers C.L., Zhu J., Kumar V., Schadt E.E.</td>
            <td>An integrative multi-network and multi-classifier approach to predict genetic interactions</td>
            <td>2010</td>
            <td>PLoS Computational Biology</td>
            <td>10.1371/journal.pcbi.1000928</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Costanzo M., Baryshnikova A., Bellay J., Kim Y., Spear E.D., Sevier C.S., Ding H., Koh J.L.Y., Toufighi K., Mostafavi S., Prinz J., St. Onge R.P., Vandersluis B., Makhnevych T., Vizeacoumar F.J., Alizadeh S., Bahr S., Brost R.L., Chen Y., Cokol M., Deshpande R., Li Z., Lin Z.-Y., Liang W., Marback M., Paw J., Luis B.-J.S., Shuteriqi E., Tong A.H.Y., Van Dyk N., Wallace I.M., Whitney J.A., Weirauch M.T., Zhong G., Zhu H., Houry W.A., Brudno M., Ragibizadeh S., Papp B., Pál C., Roth F.P., Giaever G., Nislow C., Troyanskaya O.G., Bussey H., Bader G.D., Gingras A.-C., Morris Q.D., Kim P.M., Kaiser C.A., Myers C.L., Andrews B.J., Boone C.</td>
            <td>The genetic landscape of a cell</td>
            <td>2010</td>
            <td>Science</td>
            <td>10.1126/science.1180823</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Atluri G., Bellay J., Pandey G., Myers C., Kumar V.</td>
            <td>Discovering coherent value bicliques in genetic interaction data</td>
            <td>2010</td>
            <td>Proceedings of the ACM SIGKDD International Conference on Knowledge Discovery and Data Mining</td>
            <td> </td>
            <td>Conference Paper</td>
            </tr>
            <tr>
            <td>Huttenhower C., Myers C.L., Hibbs M.A., Troyanskaya O.G.</td>
            <td>Computational analysis of the yeast proteome: Understanding and exploiting functional specificity in genomic data</td>
            <td>2009</td>
            <td>Methods in Molecular Biology</td>
            <td>10.1007/978-1-59745-540-4_15</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Pandey G., Atluri G., Steinbach M., Myers C.L., Kumar V.</td>
            <td>An association analysis approach to biclustering</td>
            <td>2009</td>
            <td>Proceedings of the ACM SIGKDD International Conference on Knowledge Discovery and Data Mining</td>
            <td>10.1145/1557019.1557095</td>
            <td>Conference Paper</td>
            </tr>
            <tr>
            <td>Koh J.L.Y., Ding H., Costanzo M., Baryshnikova A., Toufighi K., Bader G.D., Myers C.L., Andrews B.J., Boone C.</td>
            <td>DRYGIN: A database of quantitative genetic interaction networks in yeast</td>
            <td>2010</td>
            <td>Nucleic Acids Research</td>
            <td>10.1093/nar/gkp820</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Myers C.L., Chiriac C., Troyanskaya O.G.</td>
            <td>Discovering biological networks from diverse functional genomic data.</td>
            <td>2009</td>
            <td>Methods in molecular biology (Clifton, N.J.)</td>
            <td>10.1007/978-1-60761-175-2_9</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Bellay J., Vandersssluis B., Kim Y., Bandyopadhyay S., Myers C.L.</td>
            <td>Characterization of systems-level organization through mining large-scale genetic interaction networks</td>
            <td>2009</td>
            <td>2009 IEEE International Workshop on Genomic Signal Processing and Statistics, GENSIPS 2009</td>
            <td>10.1109/GENSIPS.2009.5174329</td>
            <td>Conference Paper</td>
            </tr>
            <tr>
            <td>Huttenhower C., Hibbs M.A., Myers C.L., Caudy A.A., Hess D.C., Troyanskaya O.G.</td>
            <td>The impact of incomplete knowledge on evaluation: An experimental benchmark for protein function prediction</td>
            <td>2009</td>
            <td>Bioinformatics</td>
            <td>10.1093/bioinformatics/btp397</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Pandey G., Myers C.L., Kumar V.</td>
            <td>Incorporating functional inter-relationships into protein function prediction algorithms</td>
            <td>2009</td>
            <td>BMC Bioinformatics</td>
            <td>10.1186/1471-2105-10-142</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Hibbs M.A., Myers C.L., Huttenhower C., Hess D.C., Li K., Caudy A.A., Troyanskaya O.G.</td>
            <td>Directing experimental biology: A case study in mitochondrial biogenesis</td>
            <td>2009</td>
            <td>PLoS Computational Biology</td>
            <td>10.1371/journal.pcbi.1000322</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Hess D.C., Myers C., Huttenhower C., Hibbs M.A., Hayes A.P., Paw J., Clore J.J., Mendoza R.M., Luis B.S., Nislow C., Giaever G., Costanzo M., Troyanskaya O.G., Caudy A.A.</td>
            <td>Computationally driven, quantitative experiments discover genes required for mitochondrial biogenesis</td>
            <td>2009</td>
            <td>PLoS Genetics</td>
            <td>10.1371/journal.pgen.1000407</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Dixon S.J., Fedyshyn Y., Koh J.L.Y., Prasad T.S.K., Chahwan C., Chua G., Toufighi K., Baryshnikova A., Hayles J., Hoe K.-L., Kim D.-U., Park H.-O., Myers C.L., Pandey A., Durocher D., Andrewsa B.J., Boone C.</td>
            <td>Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes</td>
            <td>2008</td>
            <td>Proceedings of the National Academy of Sciences of the United States of America</td>
            <td>10.1073/pnas.0806261105</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Guan Y., Myers C.L., Lu R., Lemischka I.R., Bult C.J., Troyanskaya O.G.</td>
            <td>A genomewide functional network for the laboratory mouse</td>
            <td>2008</td>
            <td>PLoS Computational Biology</td>
            <td>10.1371/journal.pcbi.1000165</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Guan Y., Myers C.L., Hess D.C., Barutcuoglu Z., Caudy A.A., Troyanskaya O.G.</td>
            <td>Predicting gene function in a hierarchical context with an ensemble of classifiers</td>
            <td>2008</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2008-9-s1-s3</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Peña-Castillo L., Tasan M., Myers C.L., Lee H., Joshi T., Zhang C., Guan Y., Leone M., Pagnani A., Kim W.K., Krumpelman C., Tian W., Obozinski G., Qi Y., Mostafavi S., Lin G.N., Berriz G.F., Gibbons F.D., Lanckriet G., Qiu J., Grant C., Barutcuoglu Z., Hill D.P., Warde-Farley D., Grouios C., Ray D., Blake J.A., Deng M., Jordan M.I., Noble W.S., Morris Q., Klein-Seetharaman J., Bar-Joseph Z., Chen T., Sun F., Troyanskaya O.G., Marcotte E.M., Xu D., Hughes T.R., Roth F.P.</td>
            <td>A critical assessment of Mus musculus gene function prediction using integrated genomic evidence</td>
            <td>2008</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2008-9-s1-s2</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Hibbs M.A., Hess D.C., Myers C.L., Huttenhower C., Li K., Troyanskaya O.G.</td>
            <td>Exploring the functional landscape of gene expression: Directed search of large microarray compendia</td>
            <td>2007</td>
            <td>Bioinformatics</td>
            <td>10.1093/bioinformatics/btm403</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Myers C.L., Troyanskaya O.G.</td>
            <td>Context-sensitive data integration and prediction of biological networks</td>
            <td>2007</td>
            <td>Bioinformatics</td>
            <td>10.1093/bioinformatics/btm332</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Huttenhower C., Flamholz A.I., Landis J.N., Sahi S., Myers C.L., Olszewski K.L., Hibbs M.A., Siemers N.O., Troyanskaya O.G., Coller H.A.</td>
            <td>Nearest Neighbor Networks: Clustering expression data based on gene neighborhoods</td>
            <td>2007</td>
            <td>BMC Bioinformatics</td>
            <td>10.1186/1471-2105-8-250</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Miller D.L., Myers C.L., Rickards B., Coller H.A., Jane S.J.</td>
            <td>Adenovirus type 5 exerts genome-wide control over cellular programs governing proliferation, quiescence, and survival</td>
            <td>2007</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2007-8-4-r58</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Huttenhower C., Hibbs M., Myers C., Troyanskaya O.G.</td>
            <td>A scalable method for integration and functional analysis of multiple microarray datasets</td>
            <td>2006</td>
            <td>Bioinformatics</td>
            <td>10.1093/bioinformatics/btl492</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Reguly T., Breitkreutz A., Boucher L., Breitkreutz B.-J., Hon G.C., Myers C.L., Parsons A., Friesen H., Oughtred R., Tong A., Stark C., Ho Y., Botstein D., Andrews B., Boone C., Troyanskya O.G., Ideker T., Dolinski K., Batada N.N., Tyers M.</td>
            <td>Comprehensive curation and analysis of global interaction networks in Saccharomyces cerevisiae</td>
            <td>2006</td>
            <td>Journal of Biology</td>
            <td>10.1186/jbiol36</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Sealfon R.S.G., Hibbs M.A., Huttenhower C., Myers C.L., Troyanskaya O.G.</td>
            <td>GOLEM: An interactive graph-based gene-ontology navigation and analysis tool</td>
            <td>2006</td>
            <td>BMC Bioinformatics</td>
            <td>10.1186/1471-2105-7-443</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Myers C.L., Barrett D.R., Hibbs M.A., Huttenhower C., Troyanskaya O.G.</td>
            <td>Finding function: Evaluation methods for functional genomic data</td>
            <td>2006</td>
            <td>BMC Genomics</td>
            <td>10.1186/1471-2164-7-187</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Brown J.A., Sherlock G., Myers C.L., Burrows N.M., Deng C., Wu H.I., McCann K.E., Troyanskaya O.G., Brown J.M.</td>
            <td>Global analysis of gene function in yeast by quantitative phenotypic profiling</td>
            <td>2006</td>
            <td>Molecular Systems Biology</td>
            <td>10.1038/msb4100043</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Myers C.L., Robson D., Wible A., Hibbs M.A., Chiriac C., Theesfeld C.L., Dolinski K., Troyanskaya O.G.</td>
            <td>Discovery of biological networks from diverse functional genomic data</td>
            <td>2005</td>
            <td>Genome Biology</td>
            <td>10.1186/gb-2005-6-13-r114</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Myers C.L., Chen X., Troyanskaya O.G.</td>
            <td>Visualization-based discovery and analysis of genomic aberrations in microarray data</td>
            <td>2005</td>
            <td>BMC Bioinformatics</td>
            <td>10.1186/1471-2105-6-146</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Myers C.L., Dunham M.J., Kung S.Y., Troyanskaya O.G.</td>
            <td>Accurate detection of aneuploidies in array CGH and gene expression microarray data</td>
            <td>2004</td>
            <td>Bioinformatics</td>
            <td>10.1093/bioinformatics/bth440</td>
            <td>Article</td>
            </tr>
            <tr>
            <td>Kung S.Y., Myers C.L., Zhang X.</td>
            <td>A recursive QR approach to semi-blind equalization of time-varying MIMO channels</td>
            <td>2004</td>
            <td>ICASSP, IEEE International Conference on Acoustics, Speech and Signal Processing - Proceedings</td>
            <td> </td>
            <td>Conference Paper</td>
            </tr>
            <tr>
            <td>Zhang X., Myers C.L., Kung S.Y.</td>
            <td>Cross-weighted fisher discriminant analysis for visualization of DNA microarray data</td>
            <td>2004</td>
            <td>ICASSP, IEEE International Conference on Acoustics, Speech and Signal Processing - Proceedings</td>
            <td> </td>
            <td>Conference Paper</td>
            </tr>
        </tbody>
        </table>
            </div>
            </div>
          <script>
          $(document).ready(function(){
              $('#example').dataTable({
                "iDisplayLength": 20,
                "order": [2],
                "columnDefs": [
                { "width": "30%", "targets": 1}
                ]
                });
          });
          </script>
          <script type="text/javascript" src="/js/footer.js"></script>
    </body>
</html>
