<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Research</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/modern-business.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">



</head>

<body>

    <!-- Navigation -->
    <script type="text/javascript" src="js/header.js"></script>

    <!-- Page Content -->
    <div class="container">

        <!-- Page Heading/Breadcrumbs -->
        <div class="row">
            <div class="col-lg-12">
                <h1 class="page-header">CSBIO Research
                </h1>
            </div>
        </div>
        <!-- /.row -->

        <!-- Project One -->
        <div class="row">
            <div class="col-md-5">
            <br>
            <br> 
            <br> 
                <a href="images/functional_distribution.png">
                    <img class="img-responsive img-hover" src="images/functional_distribution.png" alt=" " style="max-height: 300px">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Ultra high-throughput chemical genomics in yeast</h3>
                <p>The current inability of “target-centric” approaches to discover compounds’ molecular targets in a high-throughput manner inhibits progress in fields such as therapeutic development, toxicology screening, and biological probe discovery. With our collaborators in the Boone Lab at the University of Toronto and at RIKEN in Japan, we have developed an ultra high-throughput yeast chemical-genomics assay that allows the prediction of a compound’s gene- and process-level targets across the entire genome, filling a critical gap in the way compounds are screened for bioactivity. This methodology was applied to screen more than 13,000 compounds with diverse origins (synthetic, natural product and derivative, and clinically-relevant compounds). From these data, we identified compounds with novel targets as well as the general cellular functions that tend to be disrupted by the compounds from diverse collections. We obtain high confidence process-level predictions for over 10% of the screened compounds. At the current level of throughput, we can screen more than 10,000 compounds and generate genome-wide target predictions within a few months’ time, demonstrating that we have developed an efficient, high-throughput method to assess genome-wide bioactivities present in large compound collections.</p>
            </div>
        </div>

        <!-- /.row -->
        <hr>
        <!-- Project Two -->
        <div class="row">
            <div class="col-md-5">
                <br> 
                <br> 
                <br> 
                <br> 
                <a href="images/pathway_interaction.png">
                    <img class="img-responsive img-hover" src="images/pathway_interaction.png" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Genetic interactions in human complex diseases</h3>
                <p>Genetic interactions have been reported to underlie phenotypes in a variety of systems, but the extent to which they contribute to complex disease in humans remains unclear. In principle, genome-wide association studies (GWAS) provide a platform for detecting genetic interactions, but existing methods for identifying them from GWAS data tend to focus on testing individual locus pairs, which undermines statistical power. We are working on leveraging insights from genetic interaction screens in model organism to develop methods to discover interactions from human population genetic data.</p>
            </div>
        </div>

        <!-- /.row -->
        <hr>
        <!-- Project Three -->
        <div class="row">
            <div class="col-md-5">
                <a href="images/CoexNetwork.png">
                    <img class="img-responsive img-hover" src="images/CoexNetwork.png" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Co-expression networks</h3>
                <p> Similar expression profiles between genes (co-expression) can be used as a way to infer functional relationships between sets of highly co-expressed genes. Our research focuses on using a combination of existing and new methods that leverage co-expression networks to discover genes underlying phenotypes of quantitative traits.</p>
                <a class="btn btn-primary" href="#">View Project</i></a>
            </div>
        </div>

        <!-- Project Four -->
        <!-- /.row -->
        <hr>
        <div class="row">
            <div class="col-md-5">
                <a href="">
                    <img class="img-responsive img-hover" src="" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Pleiotropy</h3>
                <p> Pleiotropy is the phenomenon in which a single genetic locus has multiple phenotypic effects.  Phenotypes have traditionally been characteristics observed at the organism level, such as morphological traits.  But with advancing knowledge and experimental technology, perspectives on pleiotropy have expanded in number and refined in definition.  For example, a genetic unit may be a mutation, a SNP, or a gene; a phenotype may be level of a transcript or protein, survival under an environmental stress, or a curated gene function.  We take the perspective that genes work in complex and interconnected modules, so rather than counting discrete phenotypes, we treat gene pleiotropy as a network-derived characteristic.
                <br>
                <br>
                Genetic interactions (GI) of yeast, detected genome-wide with SGA, underlie a valuable and novel view of pleiotropy for a number of reasons. They are known to appear both within and between pathways, even capturing functional relationships between genes that operate in different high-level processes.  Genetic interactions recover functions that are buffered by other genes—therefore not visible in single mutants—and can reflect any biological process that affects fitness.
                <br>
                <br>
                We use modular structures in the yeast genetic interaction network, discovered with frequent itemset mining, to measure the behavior of genes in terms of their influence on high-level biological processes.  A gene whose GI network connections broadly fall under one or two processes has a focused influence on cellular functions.  A gene whose GI network connections span many processes has a broad influence on the cell.  These correspond to low and high pleiotropy, respectively.</p>
                <a class="btn btn-primary" href="#">View Project</i></a>
            </div>
        </div>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Footer -->
    <script type="text/javascript" src="js/footer.js"></script>

</body>

</html>
