<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Software</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/modern-business.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- Navigation bar is from a script I made that pastes text -->
    <script type="text/javascript" src="js/header.js"></script>

    <!-- Page Content -->
    <div class="container">

        <!-- Page Heading/Breadcrumbs -->
        <div class="row">
            <div class="col-lg-12">
                <h1 class="page-header">Software
                    <small>CSBIO</small>
                </h1>
            </div>
        </div>
        <!-- /.row -->
        
        <!-- Project One -->
        <div class="row">
            <div class="col-md-3">
                <a href="https://github.com/csbio/">
                <br>
                <br>
                <br>
                <br>
                    <img class="img-responsive img-hover" src="/images/Software/GitHub-Mark.png" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>Github</h3>
                <p>Our lab GitHub page where we keep our code
                </p>
                <a class="btn btn-primary" href="https://github.com/csbio/">View Project</a>
            </div>
        <hr>
        <div class="row">
            <div class="col-md-3">
                <a href="/software.html">
                    <img class="img-responsive img-hover" src="/images/Network.jpg" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>neXus</h3>
                <h4>Network - cross(X)-species - Search</h4>
                <p>neXus is active subnetwork search algorithm that has been designed to discover conserved subnetworks in the functional linkage networks that are enriched for active genes (such as high foldchange genes in microarray). The algorithm associates statistical significance to each subnetwork that is indicative of how possible such a clustering of genes is when the neXus is run on randomly set of active genes. In the paper [1], the algorithm was used on human and mouse microarray expression data of stem cell with respect to differentiating cells to discover conserved subnetworks representative of stem cell functions in the two species. Variations of the algorithm have been used to discover species specific subnetworks and on single species context. Python implementation of the algorithm is available at http://csbio.cs.umn.edu/neXus/help.html.
                </p>

                <p>Associated publication: [1] Deshpande R, Sharma S, Verfaillie CM, Hu W, Myers CL: A Scalable Approach for Discovering Conserved Active Subnetworks across Species. PLoS Comput Biol 2010 Dec;6(12):e1001028. http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1001028</p>
                <a class="btn btn-primary" href="http://csbio.cs.umn.edu/neXus/help.html">View Project</a>
            </div>
        </div>
        <!-- /.row -->
            
        <!-- Project One -->
        <div class="row">
            <div class="col-md-3">
                <a href="http://mosaic.cs.umn.edu">
                <br>
                <br>
                <br>
                <br>
                    <img class="img-responsive img-hover" src="/images/Software/mosaic.png" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>Mosaic</h3>
                <h4>Mechanism Of Action In Compounds</h4>
                <p>The MOSAIC database is a queryable web interface for browsing chemical genetic interaction data, molecular target predictions based on genetic interactions, Gene Ontology biological process predictions, chemical genetic profile similarity and compound structural similarity. Compounds, genes, and Gene Ontology terms are queryable using MOSAIC. The MOSAIC database is generated from publicly released data from the yeast chemical genomics project
                </p>
                <a class="btn btn-primary" href="http://mosaic.cs.umn.edu">View Project</a>
            </div>
        <hr>
        <div class="row">
            <div class="col-md-3">
                <a href="/software.html">
                    <img class="img-responsive img-hover" src="/images/Network.jpg" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>neXus</h3>
                <h4>Network - cross(X)-species - Search</h4>
                <p>neXus is active subnetwork search algorithm that has been designed to discover conserved subnetworks in the functional linkage networks that are enriched for active genes (such as high foldchange genes in microarray). The algorithm associates statistical significance to each subnetwork that is indicative of how possible such a clustering of genes is when the neXus is run on randomly set of active genes. In the paper [1], the algorithm was used on human and mouse microarray expression data of stem cell with respect to differentiating cells to discover conserved subnetworks representative of stem cell functions in the two species. Variations of the algorithm have been used to discover species specific subnetworks and on single species context. Python implementation of the algorithm is available at http://csbio.cs.umn.edu/neXus/help.html.
                </p>

                <p>Associated publication: [1] Deshpande R, Sharma S, Verfaillie CM, Hu W, Myers CL: A Scalable Approach for Discovering Conserved Active Subnetworks across Species. PLoS Comput Biol 2010 Dec;6(12):e1001028. http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1001028</p>
                <a class="btn btn-primary" href="http://csbio.cs.umn.edu/neXus/help.html">View Project</a>
            </div>
        </div>
        <!-- /.row -->

        <hr>

        <!-- Project Two -->
        <div class="row">
            <div class="col-md-3">
                <a href="http://function.princeton.edu/GRIFn/">
                    <img class="img-responsive img-hover" src="/images/Software/GRIFn.jpg" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>GRIFn</h3>
                <h4>Gene Relationship Identification in Functional data</h4>
                <p>GRIFn is a system for evaluation of datasets and methods using a functional genomics gold standard based on curation by expert biolgists. It allows users to assess the ability of their datasets or methods to recapitulate known biology both in a global sense and in the context of specific biological processes. GRIFn allows enables fair comparisons between various data types and methods.
                </p>
                <p>
                Myers CL, Barrett D, Hibbs MA, Huttenhower C, Troyanskaya OG: Finding function: evaluation methods for functional genomic data. BMC Genomics 2006, 7:187</p>
                <a class="btn btn-primary" href="http://function.princeton.edu/GRIFn/">View Project</a>
            </div>
        </div>
        <!-- /.row -->

        <hr>

        <!-- Project Three -->
        <div class="row">
            <div class="col-md-3">
                <a href="http://imp.princeton.edu/">
                    <img class="img-responsive img-hover" src="/images/Software/bioPIXIE.jpg" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>bioPIXIE</h3>
                <h4>Biological Pathway Inference from eXperimental Interaction Evidence</h4>
                <p>bioPIXIE is a novel system for biological data integration and visualization for S. cerevisiae. It allows the user to discover interaction networks and pathways in which the user's gene(s) of interest participate. The system is based on a Bayesian algorithm for identification of biological networks based on integrated diverse genomic data.
                </p>
                <p>
                Myers CL, Robson D, Wible A, Hibbs M, Chiriac C, Theesfeld CL, Dolinski K, Troyanskaya OG: Discovery of biological networks from diverse functional genomic data. Genome Biology 2005, 6(13):R114.
                </p>
                <a class="btn btn-primary" href="http://imp.princeton.edu/">View Project</a>
            </div>
        </div>
        <!-- /.row -->

        <hr>

        <!-- Project Four -->
        <div class="row">

            <div class="col-md-3">
                <a href="http://function.princeton.edu/ChARMview/">
                    <img class="img-responsive img-hover" src="/images/Software/ChARMview.jpg" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>ChARMview</h3>
                <h4>Chromosomal Abberation Region Miner and Viewer</h4>
                <p>
                    ChARMView is a visualization and analysis system for guided discovery of chromosomal abnormalities from microarray data. Our system facilitates manual or automated discovery of aneuploidies through dynamic visualization and integrated statistical analysis. ChARMView can be used with array CGH and gene expression microarray data, and multiple experiments can be viewed and analyzed simultaneously.
                </p>

                <p>Myers CL, Chen X, Troyanskaya OG: Visualization-based discovery and analysis of genomic abberations in microarray data. BMC Bioinformatics, 6:146, 2005.
                </p>

                <p>Myers CL, Dunham M, Kung SY, Troyanskaya OG: Accurate detection of aneuploidies in array CGH and gene expression microarray data. Bioinformatics, 20:3533-3543, 2004.
                </p>
                <a class="btn btn-primary" href="http://function.princeton.edu/ChARMview/">View Project</a>
            </div>
        </div>
        <!-- /.row -->

        <hr>

        <!-- Project Five -->
        <div class="row">
            <div class="col-md-3">
                <a href="#">
                    <img class="img-responsive img-hover" src="/images/Software/GOLEM.jpg" alt="">
                </a>
            </div>
            <div class="col-md-9">
                <h3>GOLEM</h3>
                <h4>Gene Ontology Local Exploration Map</h4>
                <p>GOLEM is a tool for viewing, navigating, and analyzing the hierarchical structure and annotations to the gene ontology. The visualization component allows a user to see the local graph structure around a GO term of interest and navigate to nearby nodes. GOLEM also provides the ability to look for statistical enrichment of GO terms in lists of genes and then observe the relationships between those terms. GOLEM is available both as an applet for use online and as a standalone download.</p>
                <p>
                Sealfon RSG, Hibbs MA, Huttenhower C, Myers CL, Troyanskaya OG: GOLEM: an interactive graph-based gene ontology navigation and analysis tool. BMC Bioinformatics 2006, 7:443.</p>
                <a class="btn btn-primary" href="#">View Project</a>
            </div>
        </div>
        <!-- /.row -->

        <hr>

        <!-- Pagination -->
        <div class="row text-center">
            <div class="col-lg-12">
                <ul class="pagination">
                    <li>
                        <a href="#">&laquo;</a>
                    </li>
                    <li class="active">
                        <a href="#">1</a>
                    </li>
                    <li>
                        <a href="#">2</a>
                    </li>
                    <li>
                        <a href="#">3</a>
                    </li>
                    <li>
                        <a href="#">4</a>
                    </li>
                    <li>
                        <a href="#">5</a>
                    </li>
                    <li>
                        <a href="#">&raquo;</a>
                    </li>
                </ul>
            </div>
        </div>
        <!-- /.row -->

        <hr>

    <!-- Footer -->
    <script type="text/javascript" src="js/footer.js"></script>
    </div>
    <!-- /.container -->

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>


</body>

</html>
