<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Lab Member Area</title>

    <!-- Bootstrap Core CSS -->
    <link href="../css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="../css/modern-business.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="../font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">


</head>

<body>

    <!-- Navigation bar is from a script I made that pastes text -->
    <script type="text/javascript" src="/js/header.js"></script>


    <!-- Page Content -->
    <div class="container">

        <!-- Page Heading/Breadcrumbs -->
        <div class="row">
            <div class="col-lg-12">
                <h1 class="page-header">CSBIO Research
                </h1>
            </div>
        </div>
        <!-- /.row -->

        <!-- Project One -->
        <div class="row">
            <div class="col-md-5">
            <br>
            <br> 
            <br> 
                <a href="/images/functional_distribution.png">
                    <img class="img-responsive img-hover" src="images/human_GI_crsipr.pdf" alt=" " style="max-height: 300px">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Genetic interaction analysis in human cells</h3>
                <p>We work towards a global reference map of human gene function. In model organisms, genetic interaction mapping and network analysis can elucidate the function for every gene in the genome. Together with our collaboration partner from the Moffat, Boone and Andrews labs at the Donnelly Centre for Cellular and Biomolecular Research in Toronto, we generate a genome-wide map of genetic interactions in a human cell line using CRISPR/Cas9 screening. This technology enables us to perform simultaneous mutations in two genes across millions of gene pairs. We develop and apply methods to predict which gene pairs have the biggest chance of interacting. We also develop methods to identify genetic interactions from CRISPR/Cas9 screening data and to discover a global structure in this data.</p>
            </div>
        </div>

        <!-- /.row -->
        <hr>
        <!-- Project Two -->
        <div class="row">
            <div class="col-md-5">
                <br> 
                <br> 
                <br> 
                <br> 
                <a href="/images/pathway_interaction.png">
                    <img class="img-responsive img-hover" src="images/pathway_interaction.png" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Genetic interactions in human complex diseases</h3>
                <p>Genetic interactions have been reported to underlie phenotypes in a variety of systems, but the extent to which they contribute to complex disease in humans remains unclear. In principle, genome-wide association studies (GWAS) provide a platform for detecting genetic interactions, but existing methods for identifying them from GWAS data tend to focus on testing individual locus pairs, which undermines statistical power. We are working on leveraging insights from genetic interaction screens in model organism to develop methods to discover interactions from human population genetic data.</p>
            </div>
        </div>

        <!-- /.row -->
        <hr>
        <!-- Project Three -->
        <div class="row">
            <div class="col-md-5">
                <a href="/images/SoyNetwork.png">
                    <img class="img-responsive img-hover" src="images/SoyNetwork.png" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Co-expression networks</h3>
                <p> Soybean is a major crop in the United States, used mainly for oil, protein, and animal feed. Soybean domestication and modern breeding has resulted in a very narrow germplasm that limits yield increases from year to year. Genome-wide association studies has sparked recent interest due to its ability to identify regions influencing quantitative traits but rarely lead to candidate gene identification. Similarly, the University of Minnesota recently developed a fast neutron population as a way to introduce genetic diversity which resulted in a wide variety of phenotypes. Unfortunately, there are a large number of mutations within these lines making it difficult to pinpoint genes that are causing these phenotypes. One way to combat these limitations is to utilize expression data to generate gene expression profiles. Similar expression profiles between genes (co-expression) can be used as a way to infer functional relationships between sets of highly co-expressed genes. Our research focuses on using a combination of existing and new methods that leverage co-expression networks to discover genes of interest associated with phenotypes of quantitative traits.</p>
            </div>
        </div>

        <!-- Project Four -->
        <!-- /.row -->
        <hr>
        <div class="row">
            <div class="col-md-5">
                <a href="">
                    <img class="img-responsive img-hover" src="" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Pleiotropy</h3>
                <p> Pleiotropy is the phenomenon in which a single genetic locus has multiple phenotypic effects.  Phenotypes have traditionally been characteristics observed at the organism level, such as morphological traits.  But with advancing knowledge and experimental technology, perspectives on pleiotropy have expanded in number and refined in definition.  For example, a genetic unit may be a mutation, a SNP, or a gene; a phenotype may be level of a transcript or protein, survival under an environmental stress, or a curated gene function.  We take the perspective that genes work in complex and interconnected modules, so rather than counting discrete phenotypes, we treat gene pleiotropy as a network-derived characteristic.
                <br>
                <br>
                Genetic interactions (GI) of yeast, detected genome-wide with SGA, underlie a valuable and novel view of pleiotropy for a number of reasons. They are known to appear both within and between pathways, even capturing functional relationships between genes that operate in different high-level processes.  Genetic interactions recover functions that are buffered by other genes—therefore not visible in single mutants—and can reflect any biological process that affects fitness.
                <br>
                <br>
                We use modular structures in the yeast genetic interaction network, discovered with frequent itemset mining, to measure the behavior of genes in terms of their influence on high-level biological processes.  A gene whose GI network connections broadly fall under one or two processes has a focused influence on cellular functions.  A gene whose GI network connections span many processes has a broad influence on the cell.  These correspond to low and high pleiotropy, respectively.</p>
                <!--<a class="btn btn-primary" href="#">View Project</i></a>-->
            </div>
        </div>

        <hr>
        <div class="row">
            <div class="col-md-5">
                <a href="">
                    <img class="img-responsive img-hover" src="" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Yeast genetic interactions</h3>
                <p></p>
                <!--<a class="btn btn-primary" href="#">View Project</i></a>-->
            </div>
        </div>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Footer -->
    <script type="text/javascript" src="js/footer.js"></script>

</body>

</html>
