<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>TEST AREA</title>

    <!-- Bootstrap Core CSS -->
    <link href="../css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="../css/modern-business.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="../font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">


</head>

<body>

    <!-- Navigation bar is from a script I made that pastes text -->
    <script type="text/javascript" src="/js/header.js"></script>


    <!-- Page Content -->
    <div class="container">

        <!-- Page Heading/Breadcrumbs -->
        <div class="row">
            <div class="col-lg-12">
                <h1 class="page-header">CSBIO Research
                </h1>
            </div>
        </div>
        <!-- /.row -->

        <!-- Project One -->
        <div class="row">
            <div class="col-md-5">
            <br>
            <br> 
            <br> 
                <a href="../images/CRISPR_GI.png">
                    <img class="img-responsive img-hover" src="../images/CRISPR_GI.png" alt=" " style="max-height: 300px">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Genetic interaction analysis in human cells</h3>
                <p>We work towards a global reference map of human gene function. In model organisms, genetic interaction mapping and network analysis can elucidate the function for every gene in the genome. Together with the <a href="http://moffatlab.ccbr.utoronto.ca/index.html">Moffat</a>, <a href="http://sites.utoronto.ca/boonelab/">Boone</a> and <a href="http://sites.utoronto.ca/andrewslab/">Andrews</a> labs at the Donnelly Centre for Cellular and Biomolecular Research in Toronto, we generate a genome-wide map of genetic interactions in a human cell line. Using CRISPR/Cas9 screening, we test effects of simultaneous mutations in two genes across millions of gene pairs. To represent genetic interaction information of all possible ~360,000,000 combinations, we develop methods to predict and screen gene pairs that have the best chance of interacting. We also develop methods to identify genetic interactions from CRISPR/Cas9 screening data and integrate other genomic data to generate a global atlas of human gene function.</p>
            </div>
        </div>

        <!-- /.row -->
        <hr>
        <!-- Project Two -->
        <div class="row">
            <div class="col-md-5">
                <br> 
                <a href="../images/NetworkPic.png">
                    <img class="img-responsive img-hover" src="../images/NetworkPic.png" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Co-expression networks</h3>
                <p>Using technologies such as RNA-Seq and microarrays, it is possible to measure quantitative levels of expression throughout the genome across multiple different conditions, timepoints, tissues or samples. Based on a large collection of these measurements, it is possible to build a co-expression network where one can infer a gene’s function by “guilt by association”; leveraging the principle that genes that are highly associated with each other are more likely to share a common function. Co-expression networks are built with the purpose of discovering and characterizing highly associated groups of genes or modules to better understand various phenotypic functions.  Our lab aims to understand the biological context of quantitative traits in plant species to better aide in identifying high-value genes important to plant breeding.</p>
            </div>
        </div>

        <!-- /.row -->
        <hr>
        <!-- Project Three -->
        <div class="row">
            <div class="col-md-5">
                <a href="../images/.png">
                    <img class="img-responsive img-hover" src="../images/.png" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Chemical Genomics</h3>
                <p>The expense and time required for the creation of clinically relevant drugs has been increasing. One factor for this increase includes the high cost of lead compound failures due to efficacy and safety. Solving this problem requires looking beyond single assay screens for single target compounds towards a more comprehensive picture of the mode-of-action of compounds. With our collaborators in the Boone Lab at the University of Toronto and at RIKEN in Japan, we have developed an ultra high-throughput chemical-genomics assay that allows the prediction of a compound’s gene- and process-level targets across the entire genome, filling a critical gap in the way compounds are screened for bioactivity. This methodology was applied to screen more than 13,000 compounds within yeast with diverse origins. From these data, we identified compounds with novel targets as well as the general cellular functions that tend to be disrupted by the compounds from diverse collections. In addition to our work in yeast, we are also interested in extending our current methods to bacteria. We have applied our chemical-genomics system in Escherichia coli; screening around 1000 FDA-approved compounds. This data will be helpful to understand how we can develop novel antibiotic agents. The expense and time required for the creation of clinically relevant drugs has been increasing. One factor for this increase includes the high cost of lead compound failures due to efficacy and safety. Solving this problem requires looking beyond single assay screens for single target compounds towards a more comprehensive picture of the mode-of-action of compounds. With our collaborators in the Boone Lab at the University of Toronto and at RIKEN in Japan, we have developed an ultra high-throughput chemical-genomics assay that allows the prediction of a compound’s gene- and process-level targets across the entire genome, filling a critical gap in the way compounds are screened for bioactivity. This methodology was applied to screen more than 13,000 compounds within yeast with diverse origins. From these data, we identified compounds with novel targets as well as the general cellular functions that tend to be disrupted by the compounds from diverse collections. In addition to our work in yeast, we are also interested in extending our current methods to bacteria. We have applied our chemical-genomics system in Escherichia coli; screening around 1000 FDA-approved compounds. This data will be helpful to understand how we can develop novel antibiotic agents. </p>
            </div>
        </div>

        <!-- Project Four -->
        <!-- /.row -->
        <hr>
        <div class="row">
            <div class="col-md-5">
                <a href="">
                    <img class="img-responsive img-hover" src="" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Pleiotropy</h3>
                <p> Pleiotropy is the phenomenon in which a single genetic locus has multiple phenotypic effects.  Phenotypes have traditionally been characteristics observed at the organism level, such as morphological traits.  But with advancing knowledge and experimental technology, perspectives on pleiotropy have expanded in number and refined in definition.  For example, a genetic unit may be a mutation, a SNP, or a gene; a phenotype may be level of a transcript or protein, survival under an environmental stress, or a curated gene function.  We take the perspective that genes work in complex and interconnected modules, so rather than counting discrete phenotypes, we treat gene pleiotropy as a network-derived characteristic.
                <br>
                <br>
                Genetic interactions (GI) of yeast, detected genome-wide with SGA, underlie a valuable and novel view of pleiotropy for a number of reasons. They are known to appear both within and between pathways, even capturing functional relationships between genes that operate in different high-level processes.  Genetic interactions recover functions that are buffered by other genes—therefore not visible in single mutants—and can reflect any biological process that affects fitness.
                <br>
                <br>
                We use modular structures in the yeast genetic interaction network, discovered with frequent itemset mining, to measure the behavior of genes in terms of their influence on high-level biological processes.  A gene whose GI network connections broadly fall under one or two processes has a focused influence on cellular functions.  A gene whose GI network connections span many processes has a broad influence on the cell.  These correspond to low and high pleiotropy, respectively.</p>
                <!--<a class="btn btn-primary" href="#">View Project</i></a>-->
            </div>
        </div>

        <hr>
        <div class="row">
            <div class="col-md-5">
                <a href="">
                    <img class="img-responsive img-hover" src="" alt="">
                </a>
            </div>
            <div class="col-md-7">
                <h3>Yeast genetic interactions</h3>
                <p></p>
                <!--<a class="btn btn-primary" href="#">View Project</i></a>-->
            </div>
        </div>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Footer -->
    <script type="text/javascript" src="js/footer.js"></script>

</body>

</html>
